src/org/labkey/test/tests/issues/IssueAPITest.java

```diff
@@ -0,0 +1,280 @@
+import org.labkey.test.Locator;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.InDevelopment;
+import org.labkey.test.pages.issues.DetailsPage;
+import org.labkey.test.util.APIUserHelper;
+import org.labkey.test.util.ApiPermissionsHelper;
+import org.labkey.test.util.IssuesHelper;
+import org.labkey.test.util.PermissionsHelper;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.labkey.test.util.PasswordUtil.getUsername;
+
+@Category({InDevelopment.class})
+public class IssueAPITest extends BaseWebDriverTest
+{
+    IssuesHelper _issuesHelper = new IssuesHelper(this);
+    APIUserHelper _userHelper = new APIUserHelper(this);
+    static String ISSUES = "issues";
+    static Integer TEST_USER_ID;
+    static String TEST_USER_NAME;
+    static String TEST_USER_DISPLAY_NAME;
+
+    static Integer TEST_BUDDY_ID;
+    static String TEST_BUDDY_NAME = "testbuddy@issues.test";
+    static String TEST_BUDDY_DISPLAY_NAME;
+
+    @Override
+    protected void doCleanup(boolean afterTest) throws TestTimeoutException
+    {
+        super.doCleanup(afterTest);
+    }
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        IssueAPITest init = (IssueAPITest) getCurrentTest();
+        init.doSetup();
+    }
+
+    private void doSetup()
+    {
+        TEST_USER_NAME = getUsername();
+        TEST_USER_ID = _userHelper.getUserId(TEST_USER_NAME);
+        TEST_USER_DISPLAY_NAME = _userHelper.getDisplayNameForEmail(TEST_USER_NAME);
+        _userHelper.createUser(TEST_BUDDY_NAME);
+        TEST_BUDDY_ID = _userHelper.getUserId(TEST_BUDDY_NAME);
+        TEST_BUDDY_DISPLAY_NAME = _userHelper.getDisplayNameForEmail(TEST_BUDDY_NAME);
+
+        _containerHelper.createProject(getProjectName(), null);
+        _issuesHelper.createNewIssuesList(ISSUES, getContainerHelper());
+    }
+
+    @Before
+    public void preTest()
+    {
+        goToProjectHome();
+    }
+
+    @Test
+    public void testInsertAnIssue() throws Exception
+    {
+        String comment = "This is a test to see if we can just... insert an issue";
+        String title = "Insert Issue Test Issue";
+        IssueModel testIssue = basicIssueModel(title, comment);
+
+        IssueCommand cmd = new IssueCommand();
+        cmd.setIssue(testIssue);
+        IssueResponse response = cmd.execute(createDefaultConnection(), getProjectName());
+        assertNotNull("expect our issue response to tell us an issue ID", response.getIssueId());
+
+        var issuePage = DetailsPage.beginAt(this, getProjectName(), response.getIssueId().toString());
+        var issueComment = issuePage.getComments().get(0);
```

Comment:
Should we be using the UI to validate the inserted issue? Could we instead request the new issue using your new API command and compare the response with expected?

Reply:
switched completely to using the API now